packages/cadl-ranch-specs/http/azure/core/main.cadl

```diff
@@ -49,3 +58,101 @@ Expected response body:
 ```
 """)
 op createOrUpdate is ResourceOperations.ResourceCreateOrUpdate<User>;
```

Comment:
"createOrUpdate" does not work as expected for now.

Need to check if it relates to `content-type=application/merge-patch+json`

```
warn: Request validation failed: Body provided doesn't match expected body:
 Expected:
{ name: 'Madge' }
 Actual:
{}
```

Reply:
possible fix: https://github.com/Azure/cadl-ranch/pull/203